panda_gym/pybullet.py

```diff
@@ -129,24 +125,7 @@ def render(
         Returns:
             RGB np.ndarray or None: An RGB array if mode is 'rgb_array', else None.
         """
-        if mode is not None:
-            warnings.warn(
-                "The 'mode' argument is deprecated and will be removed in "
-                "a future version. Use the 'render_mode' argument of the constructor instead.",
-                DeprecationWarning,
-            )
-        target_position = target_position if target_position is not None else np.zeros(3)
```

Comment:
This line has to stay, setting the target position to [0,0,0], in case anyone calls `env.sim.render()` directly.